docs/testing/cart-checkout/readme.md

```diff
@@ -0,0 +1,191 @@
+<!--
+taking a look from https://make.wordpress.org/test/handbook/call-for-testing/
+note: there will be many grammar mistakes since Grammarly doesn't work here, but I will run
+this through it later when I finish.
+-->
+
+# Cart and Checkout testing plan
+
+- [Cart and Checkout testing plan](#cart-and-checkout-testing-plan)
+  * [Introduction](#introduction)
+    + [Known limitations:](#known-limitations-)
+  * [Before you start](#before-you-start)
+  * [What are we testing?](#what-are-we-testing-)
+    + [Cart Block](#cart-block)
+      - [Features to test:](#features-to-test-)
+    + [Checkout Block](#checkout-block)
+      - [Features to test:](#features-to-test--1)
+  * [Testing Checklist](#testing-checklist)
+    + [Editor experience](#editor-experience)
+    + [Shipping](#shipping)
+      - [Setup](#setup)
+    + [Payments](#payments)
+      - [Setup](#setup-1)
+    + [Coupons](#coupons)
+      - [Setup](#setup-2)
+    + [Cart Items](#cart-items)
+      - [Setup](#setup-3)
+    + [Cross-platform and browser compatibility](#cross-platform-and-browser-compatibility)
+
+## Introduction
+
+Welcome, and thank you for helping us test the Cart and Checkout blocks,
+in this document, we will outline the general checklist for how to test
+the blocks, any requirements, and expectations and feature parity, some
+will require simple coding skills, and some are straightforward, we will
+separate them.
+
+### Known limitations:
+<!-- Debating on where to put this section -->
+This is a list of all known limitation for Cart and Checkout blocks, it means
+we're aware of them, and will probably not tackle them in this first release:
+
+- Cart and Checkout blocks do not support third-party plugins that integrate with
+  regular Cart and Checkout shortcode, if you somehow see a third party plugin working
+  well, this is pure coincidence, the only exception is Stripe payment gateway.
+
+## Before you start
+
+Depending on how far you will test, there are certain requirements, in general
+you will need the following:
+
+Basic:
+- A WordPress website running WooCommerce and the ability to install a plugin and edit pages.
+
+Intermediate:
+- A code editor and/or the ability to modify plugin PHP files.
+  This could be either locally if you're hosting the code there or it could from the Plugins -> Plugin Editor
+  WordPress admin page.
+
+Advanced:
+- A locally installed version of WordPress.
+- [Node 12.16.1 and npm 6.14.4 installed](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/package.json#L149-L150).
+- Ability to edit JS source files when needed.
+
+Special Cases:
+<!-- This section should be moved to stripe payment testing -->
+To test Express payment methods there are some special requirements like
+- You need to be serving the website over https, try ephemeral websites.
+- You need to be from a supported country <!-- Nadir: not yet sure about this condition and what kind of countries are exactly supported or not. -->
```

Comment:
not yet sure about this condition and what kind of countries are exactly supported or not